app/lib/features/space/widgets/space_card.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:acter/features/space/widgets/space_card_icons.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+
+import 'package:acter/common/widgets/square_avatar.dart';
+
+class SpaceCard extends StatelessWidget {
+
+   String? displayImage;
+   String? title;
+   String? subtitle;
+
+
+   SpaceCard({
+    super.key, this.displayImage,  this.title,  this.subtitle,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        if(constraints.maxWidth < 600){
```

Comment:
We also need to change this logic for a generic ```DesktopPlatform``` const checking with ```Theme.of(context).platform```,  where ever this is applied (LayoutBuilder isn't necessary). Will be much easier to understand code :)

Reply:
Noted , I will change that . Thanks :)